src/lib/logical_query_plan/lqp_translator.cpp

```diff
@@ -11,6 +11,7 @@
 #include "abstract_lqp_node.hpp"
 #include "aggregate_node.hpp"
 #include "alias_node.hpp"
+#include "bytell_hash_map.hpp"
```

Comment:
Did you see any performance benefits with that map? We usually only use it if it has a significant benefit over `unordered_map`.